nvflare/private/fed/server/server_engine.py

```diff
@@ -120,7 +119,7 @@ def __init__(self, server, args, client_manager: ClientManager, snapshot_persist
         self.parent_conn_lock = Lock()
```

Comment:
we should separate the ServerEngine into "parent ServerEngine" and "child ServerEngine" for clarity.